### ISSUE TITLE
mount nfs in unprivileged container

### ISSUE THREAD

**Issue by benaryorg** [lxc/lxd#1047]
Is it possible to mount NFS shares into an unprivileged container?

**Comment by stgraber** [comment]
It's not currently possible. Nothing that LXD can really do about it, nfs in the upstream kernel would need userns support, after which things will just start working.

Closing this as there's nothing we can do about this, it's entirely up to the upstream kernel community.

**Comment by tych0** [comment]
On Mon, Aug 24, 2015 at 11:00:35AM -0700, Stéphane Graber wrote:
> It's not currently possible. Nothing that LXD can really do about it, nfs in the upstream kernel would need userns support, after which things will just start working.
> 
> Closing this as there's nothing we can do about this, it's entirely up to the upstream kernel community.

You could mount it on the host and then bind mount it if you really
want, though.

**Comment by benaryorg** [comment]
With "bind mount" you mean to `mount --bind /host /var/lib/lxd/…/mnt/`?

The problem is that I tried to mount a directory from the host to the container, like that, which worked perfectly fine, until I ran `lxc delete container` when the mount was still there.
All of the files got deleted.

So I wanted something which was bound to the containers runtime so it would unmount when the container shut down.
